thriftbp/headers.go

```diff
@@ -44,3 +45,17 @@ func AddClientHeader(ctx context.Context, key, value string) context.Context {
 	headers = append(headers, key)
 	return thrift.SetWriteHeaderList(ctx, headers)
 }
+
+// header gets the value of a thrift header by key
+//
+// If the value is not present we fall back to a case-insensitive check
+func header(ctx context.Context, key string) (v string, ok bool) {
+	v, ok = thrift.GetHeader(ctx, key)
+	if !ok {
+		// We fall back to a case-insensitive check so that we can have
```

Comment:
This is incorrect -- it's not a case-insensitive check, it's a lower-case check.

I think you should say
```
We fall back to checking the key that an envoy proxy will send, which is a lower-case
version of the thrift key.

For details on this workaround, see:
* <link to this PR or whatever>
* <link to the envoyproxy/envoy bug>
```